integration-tests/gradle/projects/coroutines/coroutines.diff

```diff
@@ -1,62 +1,60 @@
 diff --git a/build.gradle b/build.gradle
```

Comment:
Had to update `kotlinx.coroutines` submodule to latest `develop` (as of now), as `master` is still using Kotlin 1.6.21 which is not supported in this branch.